front/src/lib/validation/schema-utils.ts

```diff
@@ -224,6 +242,16 @@ export function removeAllNonDigits(value) {
   return value.replace(/\D/gu, "");
 }
 
+export function toNumber(value) {
+  if (value === undefined || isNaN(value)) {
```

Comment:
```suggestion
  if (isNaN(value)) {
```

`isNaN(undefined) === true`

Reply:
oh ! je ne savais pas. merci pour cet apprentissage.